src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -0,0 +1,533 @@
+<template>
+  <div>
+    <confirm-dialog ref="reassignCPDialog" attach="#addEditPersonContainer"/>
+    <v-expand-transition id="addEditPersonContainer">
```

Comment:
Not sure we have a standard, but how do you feel about line spaces in the template, separating sections/dialogs etc from one another?

Reply:
I'm in favour of whitespace between blocks of code (including markup).

Reply:
I'll add some!